aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -605,8 +611,8 @@ class AztecText : EditText, TextWatcher {
              *
              * https://material.io/guidelines/layout/metrics-keylines.html#metrics-keylines-baseline-grids
              */
-            val width = DisplayUtils.getDisplayPixelWidth(context) - DisplayUtils.dpToPx(context, 32)
-            imageGetter?.loadImage(it.source, callbacks, width)
+            val width = context.resources.displayMetrics.widthPixels - DisplayUtils.dpToPx(context, 32)
```

Comment:
I assume you mean the change from `it.source` to `it.getSource()` here. If so, it's not the same as now, the `src` attribute lives inside the Attributes member, not exposed directly as a property anymore.

OTOH, if you mean the change from `DisplayUtils.getDisplayPixelWidth(context)` to `context.resources.displayMetrics.widthPixels` then I'm not sure why is displayed in the diff but it's not part of this PR. Instead, it's part of https://github.com/wordpress-mobile/WordPress-Aztec-Android/commit/52b9cadfb96f07133f290a8c1002823861f386e7#diff-7ca71cd1f009ab8438e0901205001b14R569